src/identity/reducers/index.ts

```diff
@@ -34,6 +34,7 @@ export const initialState: CurrentIdentity = {
     type: 'free',
     accountCreatedAt: '',
     paygCreditStartDate: '',
+    isUpgradeable: false,
```

Comment:
Per AIM, `isUpgradeable` is always a `boolean`, so our default state options are only `false` or `true`. The impact of making the default redux state `false` is that, if the user's network request to /quartz/orgs/:orgId fails, they can't access the checkout page. That's probably the better idea. We don't want a PAYG user on a failed request making it to a page they're not supposed to go to. And we probably don't want a FREE user to go there either if the network hasn't been able to validate that they have an active account.

Reply:
The provider for the Checkout page re-dispatches the `/identity` (or `/quartz/me`, as applicable) thunk. But the user is only able to get to the Checkout page is `isRegionBeta` is false.

![Screen Shot 2022-09-19 at 11 49 37 AM](https://user-images.githubusercontent.com/91283923/191059216-74b02446-cfb2-4995-a759-ab748c224851.png)

Reply:
@hoorayimhelping Yes, Quartz has a similar check and will return a 405 in this case.

We've had customers complain about the presence of the `Upgrade Now` button when it shouldn't be there. For that reason, I think the default of not showing the button when we're not sure is probably the best choice.